submissions/Shevchenko-26/roman-digits/main.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"strconv"
+	"strings"
+)
+
+var digitLength = 7
+var digitWidth = 5
+
+func main() {
+	fmt.Println(os.Args)
+	if len(os.Args) != 2 && len(os.Args) != 4 {
+		log.Println("Need only one input number")
+		log.Println(os.Args)
+		os.Exit(1)
+	}
+	symbolFlag := flag.Bool("symbol", false, "set custom symbol for the art")
+	flag.Parse()
+	var symbol, number string
+	if *symbolFlag {
+		number = os.Args[3]
+		symbol = os.Args[2]
+	}else{
+		number = os.Args[1]
+	}
+	if _, err := strconv.ParseUint(number, 10, 64); err != nil {
+		log.Println("Input argument is integer non-convertible: ", err.Error())
+		os.Exit(1)
+	}
+
+	digs := prepareDigits(symbol)
+
+	sb := strings.Builder{}
+	for i := 0; i < 7; i++ {
+		for _, num := range number {
+			sb.WriteString(digs[int(num-48)][i] + " ")
```

Comment:
Such a tricky code :)) Rune math is not good practice here :)) 
Just `strings.Split()` number and use `strconv` please.))
![img](https://i.ytimg.com/vi/U6MlaIe1ljs/hqdefault.jpg)